tests/core/test_scripts.py

```diff
@@ -119,3 +122,183 @@ def test_delete_collection_raise_if_the_collection_does_not_exist(self):
+            sorting=[self.OLDEST_FIRST],
+            )
+        self.registry.storage.get_all.assert_any_call(
+            collection_id='record',
+            limit=self.BATCH_SIZE,
+            parent_id='/buckets/bucket-1/collections/collection-1',
+            pagination_rules=None,
+            sorting=[self.OLDEST_FIRST],
+            )
+        self.registry.storage.get_all.assert_any_call(
+            collection_id='record',
+            limit=self.BATCH_SIZE,
+            parent_id='/buckets/bucket-1/collections/collection-1',
+            pagination_rules=[[Filter('last_modified', 110, COMPARISON.GT)]],
+            sorting=[self.OLDEST_FIRST],
+            )
+        self.registry.storage.get_all.assert_any_call(
+            collection_id='record',
+            limit=self.BATCH_SIZE,
+            parent_id='/buckets/bucket-1/collections/collection-2',
+            pagination_rules=None,
+            sorting=[self.OLDEST_FIRST],
+            )
+        self.registry.storage.get_all.assert_any_call(
+            collection_id='record',
+            limit=self.BATCH_SIZE,
+            parent_id='/buckets/bucket-1/collections/collection-2',
+            pagination_rules=[[Filter('last_modified', 210, COMPARISON.GT)]],
+            sorting=[self.OLDEST_FIRST],
+            )
+        self.registry.storage.get_all.assert_any_call(
+            collection_id='collection',
+            limit=self.BATCH_SIZE,
+            parent_id='/buckets/bucket-1',
+            pagination_rules=[[Filter('last_modified', 200, COMPARISON.GT)]],
+            sorting=[self.OLDEST_FIRST],
+            )
+        self.registry.storage.get_all.assert_any_call(
+            collection_id='bucket',
+            limit=self.BATCH_SIZE,
+            parent_id='',
+            pagination_rules=[[Filter('last_modified', 10, COMPARISON.GT)]],
+            sorting=[self.OLDEST_FIRST],
+            )
+        self.registry.storage.update.assert_any_call(
+            collection_id='quota',
+            parent_id='/buckets/bucket-1',
+            object_id='bucket_info',
+            record={'record_count': 2, 'storage_size': 160})
+        self.registry.storage.update.assert_any_call(
+            collection_id='quota',
+            parent_id='/buckets/bucket-1/collections/collection-1',
+            object_id='collection_info',
+            record={'record_count': 1, 'storage_size': 160})
+        self.registry.storage.update.assert_any_call(
+            collection_id='quota',
+            parent_id='/buckets/bucket-1/collections/collection-2',
+            object_id='collection_info',
+            record={'record_count': 1, 'storage_size': 160})
+
+        mocked.info.assert_any_call('Bucket bucket-1, collection collection-1. Final size: 1 records, 78 bytes.')
+        mocked.info.assert_any_call('Bucket bucket-1, collection collection-2. Final size: 1 records, 79 bytes.')
+        mocked.info.assert_any_call('Bucket bucket-1. Final size: 2 records, 193 bytes.')
+
```

Comment:
LGTM.

I'm not super convinced about the utilty of the assertions on the series of calls to storage, but they may help a lot when fixing potential bugs.